### PR TITLE
chore(docs): Updated read-me with punctuation where necessary

### PR DESCRIPTION
**Pull request by Olanetsoft**

### CODE REVIEW

File: packages/gatsby-image/README.md [hunk 1]
```diff
@@ -176,8 +176,8 @@ Importing from `gatsby-image/withIEPolyfill` tells Gatsby to automatically apply
 
 There are two types of responsive images supported by gatsby-image.
 
-1. Images that have a _fixed_ width and height
-2. Images that stretch across a _fluid_ container
+1. Images that have a _fixed_ width and height.
+2. Images that stretch across a _fluid_ container.
```
Comment (AishaBlake) [inline packages/gatsby-image/README.md:180]: I'm on board with correcting punctuation in general but periods should only be added to the end of a list item if the text in that list item is a complete sentence.

**Comment by AishaBlake** [comment]
@Olanetsoft, thanks for the contribution! For future reference, PRs should always have descriptions.

File: packages/gatsby-image/README.md (hunk 1 above)
Comment (Olanetsoft) [reply packages/gatsby-image/README.md:180]: oh, thanks for the enlightenment I really appreciate it.

**Comment by Olanetsoft** [comment]
You are welcome

**Pull request merged**